test/cypress/plugins/index.ts

```diff
@@ -1,5 +1,5 @@
-module.exports = (on, config) => {
-  on('before:browser:launch', (browser = {}, launchOptions) => {
+export default on => {
+  on('before:browser:launch', (browser = { name: '' }, launchOptions: { args: string[] }) => {
```

Comment:
Not sure, but we might have some interfaces available here from cypress itself

Reply:
Indeed we do